diff -ru gnome-shell-3.36.1.old/src/st/st-icon.c gnome-shell-3.36.1/src/st/st-icon.c
--- gnome-shell-3.36.1.old/src/st/st-icon.c	2020-03-31 01:29:41.354527000 +0300
+++ gnome-shell-3.36.1/src/st/st-icon.c	2020-04-04 23:24:55.013093576 +0300
@@ -418,6 +418,12 @@
       priv->opacity_handler_id = 0;
     }
 
+  if (priv->gicon == NULL && priv->fallback_gicon == NULL)
+    {
+      g_clear_pointer (&priv->icon_texture, clutter_actor_destroy);
+      return;
+    }
+
   if (!st_widget_get_resource_scale (ST_WIDGET (icon), &resource_scale))
     return;
 
