diff -ru gnome-shell-47.0.old/meson.build gnome-shell-47.0/meson.build
--- gnome-shell-47.0.old/meson.build	2024-09-14 18:17:32.000000000 +0300
+++ gnome-shell-47.0/meson.build	2024-09-15 20:02:05.036596679 +0300
@@ -28,7 +28,7 @@
 gtk_req = '>= 4.0'
 mutter_req = '>= 47.0'
 polkit_req = '>= 0.100'
-schemas_req = '>= 47.alpha'
+schemas_req = '>= 47'
 systemd_req = '>= 246'
 ibus_req = '>= 1.5.19'
 gnome_desktop_req = '>= 40'
