diff -ru xdg-desktop-portal-gnome-47.rc.old/src/meson.build xdg-desktop-portal-gnome-47.rc/src/meson.build
--- xdg-desktop-portal-gnome-47.rc.old/src/meson.build	2024-09-02 16:30:22.000000000 +0300
+++ xdg-desktop-portal-gnome-47.rc/src/meson.build	2024-09-15 22:52:15.154299186 +0300
@@ -81,7 +81,7 @@
   dependency('gio-unix-2.0'),
   dependency('gtk4', version: '>= 4.0'),
   dependency('gtk4-unix-print', version: '>= 4.0'),
-  dependency('gsettings-desktop-schemas', version: '>= 47.alpha'),
+  dependency('gsettings-desktop-schemas', version: '>= 47'),
   dependency('gnome-desktop-4'),
   dependency('gnome-bg-4'),
   libadwaita_dep,
