diff -ru gnome-control-center-47.rc.old/meson.build gnome-control-center-47.rc/meson.build
--- gnome-control-center-47.rc.old/meson.build	2024-09-15 22:41:24.342934742 +0300
+++ gnome-control-center-47.rc/meson.build	2024-09-15 22:42:59.848247008 +0300
@@ -187,7 +187,7 @@
 gnome_rr_dep = dependency('gnome-rr-4')
 gnome_settings_dep = dependency('gnome-settings-daemon', version: '>= 41.0')
 goa_dep = dependency('goa-1.0', version: goa_req_version, fallback: 'goa')
-gsettings_desktop_dep = dependency('gsettings-desktop-schemas', version: '>= 47.alpha')
+gsettings_desktop_dep = dependency('gsettings-desktop-schemas', version: '>= 47')
 libxml_dep = dependency('libxml-2.0')
 pulse_dep = dependency('libpulse', version: pulse_req_version)
 pulse_mainloop_dep = dependency('libpulse-mainloop-glib', version: pulse_req_version)
