diff -ru gtk+-3.24.23.old/gtk/meson.build gtk+-3.24.23/gtk/meson.build
--- gtk+-3.24.23.old/gtk/meson.build	2020-06-11 05:45:39.000000000 +0300
+++ gtk+-3.24.23/gtk/meson.build	2020-09-04 21:37:33.781089244 +0300
@@ -1065,14 +1065,6 @@
   install: true
 )
 
-gtk_update_icon_cache = executable(
-  'gtk-update-icon-cache',
-  'updateiconcache.c',
-  c_args: gtk_cargs,
-  dependencies: libgtk_dep,
-  install: true
-)
-
  gtk_query_immodules = executable(
   'gtk-query-immodules-3.0',
   'queryimmodules.c',
