diff -ru vte-0.82.2.orig/meson.build vte-0.82.2/meson.build
--- vte-0.82.2.orig/meson.build	2025-11-11 01:46:56.000000000 +0300
+++ vte-0.82.2/meson.build	2025-11-29 21:44:17.609264601 +0300
@@ -703,11 +703,7 @@
   gtk3_dep = dependency('', required: false)
 endif
 
-if get_option('gtk4')
-  gtk4_dep = dependency('gtk4', version: '>=' + gtk4_req_version)
-else
-  gtk4_dep = dependency('', required: false)
-endif
+gtk4_dep = dependency('gtk4', version: '>=' + gtk4_req_version)
 
 if get_option('icu')
   icu_dep = dependency('icu-uc', version: '>=' + icu_uc_req_version)
diff -ru vte-0.82.2.orig/src/meson.build vte-0.82.2/src/meson.build
--- vte-0.82.2.orig/src/meson.build	2025-11-11 01:46:56.000000000 +0300
+++ vte-0.82.2/src/meson.build	2025-11-29 21:46:47.308805514 +0300
@@ -703,24 +703,22 @@
 
 test_units += [test_colors,]
 
-if get_option('gtk3') or get_option('gtk4')
-  test_color_lightness_sources = config_sources + color_lightness_sources + files(
-    'vtetypes.cc',
-    'vtetypes.hh',
-    'color-lightness-tests.cc',
-  )
+test_color_lightness_sources = config_sources + color_lightness_sources + files(
+  'vtetypes.cc',
+  'vtetypes.hh',
+  'color-lightness-tests.cc',
+)
 
-  test_color_lightness_deps = [
-    fmt_dep,
-    glib_dep,
-    pango_dep,
-  ]
+test_color_lightness_deps = [
+  fmt_dep,
+  glib_dep,
+  pango_dep,
+]
 
-  if get_option('gtk3')
-    test_color_lightness_deps += [gtk3_dep,]
-  elif get_option('gtk4')
-    test_color_lightness_deps += [gtk4_dep,]
-  endif
+if get_option('gtk3')
+  test_color_lightness_deps += [gtk3_dep,]
+else
+  test_color_lightness_deps += [gtk4_dep,]
 endif
 
 test_color_lightness = executable(
